src/stmt.rs

```diff
@@ -246,17 +260,19 @@ pub mod parsing {
             let eq_token: Token![=] = input.parse()?;
             let init: Expr = input.parse()?;
 
-            if input.peek(Token![else]) {
-                input.parse::<Token![else]>()?;
-                let content;
-                braced!(content in input);
-                content.call(Block::parse_within)?;
-                let verbatim = Expr::Verbatim(verbatim::between(begin, input));
-                let semi_token: Token![;] = input.parse()?;
-                return Ok(Stmt::Expr(verbatim, Some(semi_token)));
-            }
+            let else_block = if input.peek(Token![else]) {
+                let else_token = input.parse::<Token![else]>()?;
+                let expr_block = input.parse::<ExprBlock>()?;
```

Comment:
Parsing ExprBlock here is incorrect because it'd incorrectly accept attributes and labels on the else-block, which is not valid syntax in Rust. Things like: `let true = false else 'label: { return; }; let true = false else #[foo] { return; };`